test/stdlib/NSSlowTaggedLocalizedString.swift

```diff
@@ -0,0 +1,32 @@
+// RUN: mkdir -p %t
+// RUN: %target-clang -fobjc-arc %S/Inputs/NSSlowTaggedLocalizedString/NSSlowTaggedLocalizedString.m -fno-objc-arc -c -o %t/NSSlowTaggedLocalizedString.o
+// RUN: %target-build-swift -Xfrontend -disable-access-control -I %S/Inputs/NSSlowTaggedLocalizedString/ %t/NSSlowTaggedLocalizedString.o %s -o %t/a.out
+// RUN: %target-codesign %t/a.out
+// RUN: %target-run %t/a.out
+
+// REQUIRES: executable_test
+// REQUIRES: objc_interop
+
+import NSSlowTaggedLocalizedString
+import Swift
+
+import StdlibUnittest
+
+let longTaggedTests = TestSuite("NonContiguousTaggedStrings")
+
+longTaggedTests.test("EqualLongTagged") {
```

Comment:
Aside from being absolutely horrific, this is failing for me locally with this error: "error: type of expression is ambiguous without more context"

I'm completely at a loss for why that would be happening, suggestions welcome.

Reply:
Not sure if is possible, but use `-debug-constraints` or `debug-constraints-on-line=${line}` and dump the type checker debug output into a file we would be able to get a hint into what type checker is doing and where it is getting lost? Hope that helps =]

Reply:
This helped me figure it out! Thanks again :)